### DISCUSSION TITLE
Clienti privi di Fattura non visibili nella chiamata GetClient o ListClients

### DISCUSSION THREAD

**Discussion by gianlu933** [fattureincloud/api#434] [Q&A] [answered]
Buongiorno, avrei una domanda in merito ad alcuni clienti che posso vedere nella chiamata /user/companies, ma sicomme  sono clienti privi ancora di fatture non li ritrovo nelle chiamate GetClient e ListClienti. Mancandomi cosi una serie di campi che mi interessano come le informazioni sull'indirizzo, il tipo di pagamento e i metodi etc. Devo usare qualche filtro o parametro specifico ?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @gianlu933,
no, l'anagrafica clienti deve essere popolata con gli appositi metodi, come specificato qui: [Client Creation](https://developers.fattureincloud.it/docs/guides/client-creation).
La user/companies ritorna solo le aziende a cui si ha accesso con il token attuale. Mi sembra strano però che possa vedere questi clienti tramite la user/companies.

Grazie
Mauro

**Reply by valmoz** [reply]
buongiorno @gianlu933 
possiamo chiudere la discussione?

Grazie
Mauro
